config/checkstyle_checks.xml

```diff
@@ -512,6 +512,9 @@
       <property name="format" value="^(id)|([a-z][a-z0-9][a-zA-Z0-9]+)$"/>
       <property name="ignoreOverridden" value="true"/>
     </module>
+    <module name="LambdaParameterName">
+      <property name="format" value="^(id)|([a-z][a-z0-9][a-zA-Z0-9]+)$"/>
+    </module>
```

Comment:
please register your Check in google_checks.xml
and at http://checkstyle.sourceforge.net/google_style.html (see xdoc)

Please also add ITs for google style, they can be "copy" of UT.